runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,387 @@
+					u32,
+				>>::get(),
+				100,
+			);
+		}
+
+		#[test]
+		fn motion_duration_is_7_days() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::MotionDuration,
+				>(),
+				TypeId::of::<CouncilMotionDuration>(),
+			);
+
+			assert_eq!(
+				<<Runtime as pallet_collective::Config<CouncilCollective>>::MotionDuration as Get<BlockNumber>>::get(),
+				7 * DAYS,
+			);
+		}
+
+		#[test]
+		fn proposals_are_runtime_calls() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::Proposal>(),
+				TypeId::of::<RuntimeCall>(),
+			);
+		}
+
+		#[test]
+		fn set_members_origin_ensures_root() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::SetMembersOrigin,
+				>(),
+				TypeId::of::<EnsureRoot<AccountId>>(),
+			);
+		}
+
+		#[test]
+		fn default_weights_are_not_used() {
+			assert_ne!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::WeightInfo>(
+				),
+				TypeId::of::<()>(),
+			);
+		}
+	}
+
+	mod membership {
+		use super::*;
+
+		#[test]
+		fn add_origin_requires_unanimous_vote() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::AddOrigin>(
+				),
+				TypeId::of::<UnanimousCouncilVote>(),
+			);
+
+			assert_eq!(
+				TypeId::of::<UnanimousCouncilVote>(),
+				TypeId::of::<
+					EitherOfDiverse<
+						EnsureRoot<AccountId>,
+						EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn number_of_members_is_100() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::MaxMembers>(
+				),
+				TypeId::of::<CouncilMaxMembers>(),
+			);
+
+			assert_eq!(CouncilMaxMembers::get(), 100);
+		}
+
+		#[test]
+		fn council_handles_membership_changes() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_membership::Config<CouncilMembership>>::MembershipChanged,
+				>(),
+				TypeId::of::<Council>(),
+			);
+		}
+
+		#[test]
+		fn council_handles_membership_initialization() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::MembershipInitialized>(),
+				TypeId::of::<Council>(),
+			);
+		}
+
+		#[test]
+		fn prime_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::PrimeOrigin>(
+				),
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+			);
+			assert_eq!(
+				TypeId::of::<AtLeastThreeFourthsOfCouncil>(),
+				TypeId::of::<
+					EitherOfDiverse<
+						EnsureRoot<AccountId>,
+						EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 4>,
+					>,
+				>(),
+			);
+		}
+
+		#[test]
+		fn remove_origin_ensures_at_least_three_fourths() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_membership::Config<CouncilMembership>>::RemoveOrigin>(
+				),
+				TypeId::of::<AllMembersButOne>(),
+			);
```

Comment:
Based on the `AllMembersButOne`, this is inaccurate IIUC. It's 3/4 at genesis. But if we add/remove members than it's no longer 3/4

Reply:
Correct the title is not in line with the config. I think it will be better rolling [604bb64](https://github.com/r0gue-io/pop-node/pull/448/commits/604bb6456a9bf8942fa928253d245c66f06fd71c) back.

Reply:
Changes reverted: [a76aa2e](https://github.com/r0gue-io/pop-node/pull/448/commits/a76aa2ef27aea912fc272143e744046b45fe25d6)